simplecal/simplecal

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Licensed under the terms of the GNU GENERAL PUBLIC LICENSE v2, or any later version.
+
+#
```

Comment:
Avoid empty/unnecessary lines. One after the license info is good, but you don't need three surrounding each comment.

    #
    # date pattern, more at 'man date'
    #
     
    DATE_PATTERN="%H:%M:%S %d.%m.%Y"
should just be

    # see 'man date' for date patterns
    DATE_PATTERN="%H:%M:%S %d.%m.%Y"

Make similar modifications to other comment.